parsers/to-react-native/README.md

```diff
@@ -0,0 +1,150 @@
+# To React Native
+
+## Description
+
+Transform design tokens to a JavaScript `theme` object which can be used in React Native.
+
+## Usage
+
+### Options
+
+| Parameter          | Required | Type                                                                       | Default     | Description                                                                                                           |
+| ------------------ | -------- | -------------------------------------------------------------------------- | ----------- | --------------------------------------------------------------------------------------------------------------------- |
+| `colorFormat`      | optional | `rgb`, `prgb`, `hex`, `hex6`, `hex3`, `hex4`, `hex8`, `name`, `hsl`, `hsv` | `rgb`       | The format of all colors (gradients, borders, textStyles etc.)                                                        |
+| `objectName`       | optional | `string`                                                                   | `'theme'`   | The name of the JS object containing the theme,                                                                       |
+| `assetsFolderPath` | optional | `string`                                                                   | `undefined` | The relative location of the folder to import the assets from, if not provided, the assets will be referenced by URL. |
+| `prettierConfig`   | optional | [`prettier.Config`](https://prettier.io/docs/en/options.html)              | `undefined` | Formatting options of the file.                                                                                       |
+
+### Before/After
```

Comment:
Please include the example of usage inside a "Usage" section.

Also, please provide the config passed to the parser to return the mentioned output. I'vbe created the section for you but you need to update it to provide the options you used.

Here's the end result:

---

## Usage

### Config
```json
{
  "name": "to-jss",
  "options": {
  
  }
}
```

### Before/After

#### Input

```js
[
  {
    name: 'activity.svg',
    value: {
      url: 'https://s3-us-west-2.amazonaws.com/figma-alpha-api/img/a114/ce5c/947dcb83ea93c2da18ee2ea16f470a30',
    },
    type: 'vector',
  },
  {
    name: 'Body',
    value: {
      font: {
        meta: {
          source: 'localStyles',
        },
        name: 'Inter-Medium',
        type: 'font',
        value: {
          isItalic: false,
          fontFamily: 'Inter',
          fontWeight: 500,
          fontPostScriptName: 'Inter-Medium',
        },
      },
      color: {
        value: {
          a: 1,
          b: 196,
          g: 196,
          r: 196,
        },
      },
      fontSize: {
        value: {
          unit: 'px',
          measure: 14,
        },
      },
      textAlign: {
        vertical: 'top',
        horizontal: 'left',
      },
      lineHeight: {
        value: {
          unit: 'px',
          measure: 20,
        },
      },
    },
    type: 'textStyle',
  },
  {
    name: 'Colors / Accent',
    value: {
      a: 1,
      b: 239,
      g: 80,
      r: 102,
    },
    type: 'color',
  },
];
```

#### Output

```js
import acmeLogo from '../src/assets/acmeLogo.png';
import activity from '../src/assets/activity.svg';

const theme = {
  bitmap: {
    acmeLogo: acmeLogo,
  },
  vector: {
    activity: activity,
  },
  depth: {
    background: {
      elevation: 1,
      shadowOpacity: 0.1815,
      shadowRadius: 0.54,
      shadowOffset: { width: 0.6, height: 0.6 },
    },
  },
  duration: {
    base: 300,
  },
  measurement: {
    baseSpace01: 4,
  },
  textStyle: {
    body: {
      fontWeight: 500,
      fontSize: 14,
      lineHeight: 20,
      fontFamily: 'Inter-Medium',
      color: '#1e212b',
      letterSpacing: 10,
    },
  },
  border: {
    borderAccent: {
      borderWidth: 2,
      borderStyle: 'solid',
      borderColor: 'rgb(102, 80, 239)',
      borderRadius: 16,
    },
  },
  color: {
    colorsAccent: 'rgb(87, 124, 254)',
  },
  font: {
    firaCodeMedium: 'FiraCode-Medium',
  },
  gradient: {
    gradientsColored: [
      {
        angle: 90,
        colors: ['rgb(245, 72, 63)', 'rgb(255, 142, 5)'],
        locations: [0, 1],
      },
    ],
  },
  opacity: { subtle: 0.5, transparent: 0.1, visible: 0.95 },
};
export default theme;
```